source/openfisca-web-api/index.md

```diff
@@ -9,13 +9,22 @@
    config-openapi
 ```
 
-OpenFisca provides a web API package compatible with all country packages.
-Using a web interface, App Developers can access information and computations without installing anything locally.
+OpenFisca provides a web API package, compatible with all country packages.
+Using a web interface, app developers can access information and computations without installing anything locally.
```

Comment:
I'm not 100% happy with the wording "without installing anything locally". The intended meaning I infer is "without installing Python locally", but interacting with the API may require having access to tools (a browser, a CLI, a text editor) which are not universal to all platforms (e.g. a mobile phone).